bot/discord/commands/server/list.js

```diff
@@ -1,6 +1,6 @@
 const axios = require('axios')
 exports.run = async(client, message, args) => {
-    message.channel.send('Loading servers...')
+    const loadingMsg = message.channel.send('Loading servers...')
```

Comment:
`await` is required here

Reply:
Just fixed this. 👍 

https://github.com/DanBot-Hosting/DanBotHostingStats/commit/fe9cd8951bd1e39c8334c9ed352301202dbbf696